requests/__init__.py

```diff
@@ -71,12 +71,11 @@ def check_compatibility(urllib3_version, chardet_version):
     assert minor < 1
     assert patch >= 2
 
-# Check for older version of pyOpenSSL
-def __check_pyopenssl(pyopenssl_version, cryptography_version):
-    # Warning only applies for Cryptography < 1.3.4
+def _check_cryptography(cryptography_version):
+    # Cryptography < 1.3.4
```

Comment:
One petty nit and we're set here I think. Mind lowecasing the cryptography package name here, below in the warning message, and in the try/except comment?